src/systems/a320_systems_wasm/src/lib.rs

```diff
@@ -254,3 +178,225 @@ impl ElectricalBusConnection {
         }
     }
 }
+
+struct Brakes {
+    id_brake_left: sys::DWORD,
+    id_brake_right: sys::DWORD,
+    id_parking_brake: sys::DWORD,
+    id_brake_keyboard: sys::DWORD,
+    id_brake_left_keyboard: sys::DWORD,
+    id_brake_right_keyboard: sys::DWORD,
+
+    brake_left_sim_input: f64,
+    brake_right_sim_input: f64,
+    brake_left_sim_input_keyboard: f64,
+    brake_right_sim_input_keyboard: f64,
+    left_key_pressed: bool,
+    right_key_pressed: bool,
+
+    brake_left_output_to_sim: f64,
+    brake_right_output_to_sim: f64,
+
+    parking_brake_lever_is_set: bool,
+}
+impl Brakes {
+    const KEYBOARD_PRESS_SPEED: f64 = 0.6;
+    const KEYBOARD_RELEASE_SPEED: f64 = 0.3;
+
+    fn new(sim_connect: &mut Pin<&mut SimConnect>) -> Result<Self, Box<dyn std::error::Error>> {
+        Ok(Self {
+            // SimConnect inputs masking
+            id_brake_left: sim_connect
+                .map_client_event_to_sim_event("AXIS_LEFT_BRAKE_SET", true)?,
+            id_brake_right: sim_connect
+                .map_client_event_to_sim_event("AXIS_RIGHT_BRAKE_SET", true)?,
+            id_parking_brake: sim_connect.map_client_event_to_sim_event("PARKING_BRAKES", true)?,
```

Comment:
I don't know if we're aiming to support all events in our code, but if yes "PARKING_BRAKE_SET" would be missing here. (Sorry to be annoying already :D)

Reply:
I don't have this one in the list here http://www.prepar3d.com/SDKv3/LearningCenter/utilities/variables/event_ids.html
What kind of event is this?  
I think I tested to apply park brakes with keyboard and it seems it works, maybe I forget another use case there?

Reply:
In the MSFS settings you can find it under "SET PARKING BRAKE" (you probably tested with "TOGGLE PARKING BRAKE"). It's a key event with data 0 or 1. I guess the use case is for switches that give constant input (and I guess toggle would always toggle the states?). You can find it in the gauges.h in the SDK, but you're right it's not in the online documentation

Reply:
Ok so it's possible to configure a key for this one and test?  I'll try and see if it moves anything !

Reply:
Exactly :) If you want also just ping me on discord it's easier then

Reply:
Welcome to the MSFS events magic...a good check is what is available in the assignments inside the sim and check the config file afterwards what the event is named and then fingers crossed it's available via SimConnect.

Reply:
Added the event, thanks a lot!